pom.xml

```diff
@@ -107,12 +104,13 @@
     <dependency>
       <groupId>org.jboss.pnc</groupId>
       <artifactId>pnc-common</artifactId>
-      <version>2.0.0-SNAPSHOT</version>
+      <version>2.0.1-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>io.fabric8</groupId>
       <artifactId>openshift-client</artifactId>
-      <version>5.0.1</version>
+      <!-- 5.3.0: the tests fails with: java.lang.NoClassDefFoundError: io/fabric8/kubernetes/client/dsl/InOutCreateable-->
+      <version>5.2.1</version>
```

Comment:
https://quarkus.io/guides/kubernetes-client

There's also a native Quarkus extension for fabric8's openshift-client which you can `@Inject` and has configurations already setup.